test/Parse/ConditionalCompilation/decl_parse_errors.swift

```diff
@@ -31,10 +31,10 @@ class C { // expected-note 2 {{in declaration of 'C'}} expected-note {{to match
 
 #if os(iOS)
 	func foo() {}
-} // expected-error{{expected declaration}} expected-error{{expected #else or #endif at end of conditional compilation block}}
```

Comment:
I removed this error because it makes me feel that this `#if` block was *implicitly* terminated by the parser.
Probably, we should emit a specialized error for this, something like `unexpected '}' in conditional compilation block` or `conditional compilation blocks must surround self-contained statements`.
How do you think?